concepts/05 UI Components/Chart/02 Overview/05 Elements.md

```diff
@@ -0,0 +1,10 @@
+<p><img src="/images/ChartJS/chart-elements.png" alt="Chart elements" style="margin:0px auto; display:block; width: 870px;"></p>
+
+1. [Value axis](/concepts/05%20UI%20Components/Chart/20%20Axes/00%20Overview.md '/Documentation/Guide/UI_Components/Chart/Axes/Overview/')
+2. [Argument axis](/concepts/05%20UI%20Components/Chart/20%20Axes/00%20Overview.md '/Documentation/Guide/UI_Components/Chart/Axes/Overview/')
```

Comment:
I think we can highlight argument axis similar way as we highlight value axis in the picture